src/ray/raylet/scheduling/scheduling_policy.cc

```diff
@@ -29,6 +29,7 @@ int64_t HybridPolicy(const ResourceRequest &resource_request, const int64_t loca
   {
     // Make sure the local node is at the front of the list so that 1. It's first in
     // traversal order. 2. It's easy to avoid sorting it.
+    RAY_LOG(INFO) << "changes for adding local node -- starts";
```

Comment:
I think it's possible that the local node will be infeasible so we shouldn't always add it. (This would also be a good test case).